CMakePresets.json

```diff
@@ -0,0 +1,99 @@
+{
+  "version": 5,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 15,
+    "patch": 0
+  },
+  "include": [],
+  "configurePresets": [
+    {
+      "name": "windows_default",
+      "displayName": "MSVC Debug",
+      "description": "MSVC build using Visual Studio generator",
+      "generator": "Visual Studio 17 2022",
+      "binaryDir": "${sourceDir}/build/",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "CMAKE_INSTALL_PREFIX": "install",
+        "Launcher_QT_VERSION_MAJOR": "6",
+        "CMAKE_PREFIX_PATH": "${sourceDir}/libraries/Qt/6.5.1/msvc2019_64/lib/cmake",
```

Comment:
Personally I think anyone using the official QT installer is insane because it barely works with download speeds in the low tens of kilobytes. 
That said the aqt install won't interfere with the official install and a user can always make a `CmakeUserPresets.json` to override with their own config.